pkg/cmd/doctorcmd.go

```diff
@@ -351,6 +351,14 @@ func (c *Config) runDoctorCmd(cmd *cobra.Command, args []string) error {
 			versionArgs: []string{"version"},
 			versionRx:   regexp.MustCompile(`(?m)=\s*v(\d+\.\d+\.\d+)`),
 		},
+		&binaryCheck{
+			name:        "passhole-command",
+			binaryname:  c.Passhole.Command,
+			ifNotSet:    checkResultWarning,
+			ifNotExist:  checkResultInfo,
+			versionArgs: []string{"version"},
```

Comment:
The correct argument to get the version is `--version`, not `version`. Was this ever tested?